src/model_parser.h

```diff
@@ -68,15 +68,8 @@ class ModelParser {
     ENSEMBLE_SEQUENCE
   };
 
-  explicit ModelParser()
-      : inputs_(std::make_shared<ModelTensorMap>()),
-        outputs_(std::make_shared<ModelTensorMap>()),
-        composing_models_map_(std::make_shared<ComposingModelMap>()),
-        scheduler_type_(NONE), max_batch_size_(0), is_decoupled_(false),
-        response_cache_enabled_(false),
-        top_level_response_caching_enabled_(false)
-  {
-  }
+  explicit ModelParser(
+      InferenceLoadMode inference_load_mode = InferenceLoadMode::None);
```

Comment:
Same comment as [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#r2058835804). ModelParser constructor now handles creating "timestamp" input for fixed schedule mode.